app/services/core/persistent-stateful-service.ts

```diff
@@ -38,4 +45,15 @@ export abstract class PersistentStatefulService<TState extends object> extends S
       },
     );
   }
+
+  runMigrations(persistedState: any, migrations: IMigration[]) {
+    const migratedState = cloneDeep(persistedState);
+    migrations.forEach(migration => {
+      if (!persistedState[migration.oldKey]) return;
```

Comment:
`if (persistedState[migration.oldKey] == null)`
This way we properly migrate falsey values like `0`